docs/version0.40.md

```diff
@@ -42,6 +43,12 @@ In environments with low physical memory availability, the VM might use too much
 
 For more information, see [`-XX:codecachetotalMaxRAMPercentage`](xxcodecachetotalmaxrampercentage.md).
 
+### VM starts with a warning message after the container detection fails
+
+If `/proc` was mounted with the `hidepid=2` option on Linux&reg; systems, the VM was not able to access the `/proc` file system without the root privileges and therefore, the VM was exiting. Now, the VM starts with a warning message after the container detection fails.
```

Comment:
I think there also needs to be clarification that the behaviour changed in this release. And there should be more active voice. eg "If you mount `/proc` with the `hidepid=2` option on Linux&reg; systems, the VM cannot access the `/proc` file system without root privileges. In previous releases, the VM exits when it encounters this scenario. From this release, the VM ....."

"without the root privileges" is ambiguous. Do you mean the file system that doesn't have root privileges or do you mean the VM can't access the file system because it doesn't have root privileges or ...

Also, the first "proc" has no translatable noun. We don't translate these docs but there's always a possibility that might happen in the future so we should still write for translation too.